ynput/OpenPype
# Folder batch publishing tool

## Changelog Description
Batch Publisher can be opened from AYON launcher tray. 
It is a tool for batch publishing multiple files each as a separate product.

## Testing notes:
1. git clone the OpenPype repo
2. checkout this branch "Feature/op 7692 folder batch publishing tool"
3. Setup dev environment on AYON server and set OpenPype directory for bundle to directory of OpenPype
4. Ask @bradenjennings for ayon internal testing data
5. Make sure your AYON server has a project available. For example make sure you setup AYON server on AY_CG_demo project .
6. Open Batch publisher for AYON launcher tray
7. Choose the browse button and pick the root directory of test data, here
C:\errolDotting
8. The table view should show some fbx file paths
9. Choose your folder (asset) to make the line "defined"
